src/main/java/digital/slovensko/autogram/core/eforms/EFormUtils.java

```diff
@@ -347,4 +351,146 @@ public static String fillXsdIdentifier(String formIdentifier) {
 
         return "http://schemas.gov.sk/form/" + url + "/form.xsd";
     }
+
+    public static boolean isOrsrUri(String uri) {
+        return uri != null && uri.contains("://eformulare.justice.sk");
+    }
+
+    public static String getFsFormIdFromFilename(String filename) {
+        var matcher = Pattern.compile("^.+_fs([a-zA-Z0-9_-]+__\\d+__\\d+).*\\.(xml|xdcf|asice|sce|)$").matcher(filename);
+
+        if (!matcher.find())
+            return null;
+
+        return matcher.group(1);
+    }
```

Comment:
Toto by si asi zasluzilo niekde popisat (kde?), ze ako toto funguje. Bude to aj lucia potrebovat pre GoPro.

Reply:
@jsuchal pridal som niečo do README. Zoznam ID formulárov by mal vedieť user odniekiaľ získať, tak som dal odkaz na náš S3, ale možno by som na tú S3 k tomu ešte nahodil osobitný súbor, ktorý bude zoznam ID aj s nejakými aspoň názvami.

Plus ešte musím upraviť ten regex, lebo zatiaľ nechce robiť presne to, čo by som očakával.